packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -306,10 +316,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
   }
 
   public componentWillUnmount(): void {
-    super.componentWillUnmount();
-
-    // remove the eventHanlder that was added in componentDidMount
-    this._events.off(this._comboBoxWrapper.current);
```

Comment:
Is it safe to remove this?

Reply:
`EventGroup.dispose` invokes `EventGroup.off` so it should be OK

Reply:
yes calling `dispose` does the same